templates/sending_profiles.html

```diff
@@ -0,0 +1,143 @@
+			<input id="ignore_cert_errors" type="checkbox" checked>
+			<label for="ignore_cert_errors">Ignore Certificate Errors <i class="fa fa-question-circle" data-toggle="tooltip" data-placement="right" title="Ignore common certificate errors such as self-signed certs (exposes you to MiTM attacks - use carefully!)"></i></label>
+		</div>
+		<!-- <button type="button" data-toggle="modal" data-target="#sendTestEmailModal" class="btn btn-primary"><i class="fa fa-envelope"></i> Send Test Email</button> -->
+		<!-- disable sendTestEmail functionality on sending profile page until update handling of /util/send_test_email -->
+	    </div>
+        </div>
+        <div class="modal-footer">
+            <button type="button" data-dismiss="modal" class="btn btn-default" onclick="dismiss()">Cancel</button>
+            <button type="button" class="btn btn-primary" id="modalSubmit">Save Profile</button>
+        </div>
+    </div>
+  </div>
+</div>
+<!-- Send Test Email Modal -->
+<!-- <div class="modal" id="sendTestEmailModal" tabindex="-1" role="dialog" aria-labelledby="modalLabel">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content"> -->
+        <!-- New Email Modal -->
+        <!-- <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+          <h4 class="modal-title" id="sendTestEmailModalTitle">Send Test Email</h4>
+        </div>
+        <div class="modal-body">
+            <div class="row" id="sendTestEmailModal.flashes"></div>
+            <div class="row">
+                <div class="col-sm-12">
+                    <label class="control-label" for="to">Send Test Email to:</label>
+                </div>
+                <br>
+                <div class="col-sm-2">
+                    <input type="text" class="form-control" placeholder="First Name" name="to_first_name">
+                </div>
+                <div class="col-sm-2">
+                    <input type="text" class="form-control" placeholder="Last Name" name="to_last_name">
+                </div>
+                <div class="col-sm-4">
+                    <input type="email" class="form-control" placeholder="Email" name="to_email" required>
+                </div>
+                <div class="col-sm-4">
+                    <input type="text" class="form-control" placeholder="Position" name="to_position">
+                </div>
+            </div>
+        </div>
+        <div class="modal-footer">
+            <button type="button" data-dismiss="modal" class="btn btn-default">Cancel</button>
+            <button type="button" class="btn btn-primary" id="sendTestModalSubmit" onclick="sendTestEmail()"><i class="fa fa-envelope"></i> Send</button>
+        </div>
+    </div>
+  </div>
+</div> -->
+<!-- disable sendTestEmail functionality on sending profile page until update handling of /util/send_test_email -->
```

Comment:
This will require changes to `func API_Send_Test_Email` in api controller for default handling of empty template & landing page. Your call, can be worked as another issue before 0.1.2 release.